bn-models/src/models/_lookups/port.ts

```diff
@@ -6,17 +6,71 @@ export const PortTypeName = 'port';
 // TODO add an at-sea transfer Port ID to DB
 
 export interface Port extends BaseLookup {
-  name?: string;
-  code?: string;
-  group?: string;
-  state?: string;
+  name?: string;  // PORT_NAME from PORTS table
```

Comment:
My only comment is, as a newb, I don't really know what the PORTS table is, so maybe a few extra words in this first comment about where the PORT table is from? Especially since that information may end up lost in time at some point.